src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -44,6 +44,8 @@ AdaptDispatch::AdaptDispatch(ConGetSet* const pConApi,
     _fChangedMetaAttrs(false),
     _TermOutput()
 {
+    _fIsOriginModeRelative = false; // by default, the DECOM origin mode is absolute.

```

Comment:
I was following the pattern of the `_coordSavedCursor` and `_srScrollMargins` assignments, since they're somewhat related, and I thought maybe there was a reason they were done that way. I'd be happy to move these to initializer list, though.

Reply:
I can't remember which one we're trying to encourage right now. @adiviness?

Reply:
What we're doing now is to prefer that defaults for fields are set in the header, but in this case there isn't an associated header so it would be better if they were set above rather than in the constructor braces.

Reply:
I've moved the flags up into the intializer list in commit 57367b1177c371ba08557930a7f0a6fa41011342.